packages/p2p/src/components/my-profile/my-profile-stats/my-profile-name/my-profile-name.jsx

```diff
@@ -30,16 +42,81 @@ const MyProfileName = () => {
                         <Text color='prominent' weight='bold' size='s' line_height='m'>
                             {general_store.nickname}
                         </Text>
-                        <Text color='less-prominent' size={isMobile() ? 'xxxs' : 'xs'}>
-                            {joined_since > 0 ? (
-                                <Localize
-                                    i18n_default_text='Joined {{days_since_joined}}d'
-                                    values={{ days_since_joined: joined_since }}
-                                />
+                        <div className='my-profile-name--rating'>
+                            {rating_average && recommended_average ? (
+                                <React.Fragment>
+                                    <div className='my-profile-name--rating__row'>
+                                        <StarRating
+                                            empty_star_className='my-profile-name--rating--star'
+                                            empty_star_icon='IcEmptyStar'
+                                            full_star_className='my-profile-name--rating--star'
+                                            full_star_icon='IcFullStar'
+                                            initial_value={rating_average_decimal}
+                                            is_readonly
+                                            number_of_stars={5}
+                                            should_allow_hover_effect={false}
+                                            star_size={20}
+                                        />
+                                        &nbsp;
```

Comment:
Do you think we can make use of the `gap` property in `my-profile-name--rating__row` css class instead of `&nbsp` here?